### PR TITLE
Update SearchForm.vue

### PR DESCRIPTION
**Pull request by nikitamadin**
Added padding fix for long query

### CODE REVIEW

File: src/components/SearchForm.vue [hunk 1]
```diff
@@ -90,6 +90,10 @@ export default {
   .algolia-autocomplete {
     width: 100%;
   }
+
+  &__input {
+    padding-right: 2.3rem;
+  }
```
Comment (Mudassar045) [inline src/components/SearchForm.vue]: I don't think we should put this. Please avoid to use random units e.g. `2.3rem` Thanks for your efforts ! @nikitamadin

Comment (Mudassar045) [inline src/components/SearchForm.vue]: Can you just change it to `padding-right: 2rem;`?

Comment (Mudassar045) [inline src/components/SearchForm.vue]: Can you just change it to `padding-right: 2rem;` ?

Comment (nikitamadin) [reply src/components/SearchForm.vue]: @Mudassar045 no problem, I just wanted to do the same spacing to the both icon sides

**Review by thetre97 — APPROVED** [review]

**Pull request merged**